Add screenshots to showcase new UI updates

maitreyie-chavan opened the pull request:
## Summary

Recommending to replace the screenshots currently used for `Site Preview` in onboarding wizard and `Settings Page` to incorporate the new site scan tab with the new ones uploaded

### Details of Screenshots Added

| Outdated  | New  UI |
| ------------- | ------------- |
| <img width="404" alt="Screenshot 2021-12-08 at 3 31 11 PM" src="https://user-images.githubusercontent.com/20221326/145188725-1157b502-eff9-42ad-a124-9d86ee7ea2e1.png"> | <img width="677" alt="screenshot-7" src="https://user-images.githubusercontent.com/20221326/145188829-31a4195d-8bd4-4ac0-94b9-75bf85e4c1cc.png"> |
| <img width="303" alt="Screenshot 2021-12-08 at 3 30 42 PM" src="https://user-images.githubusercontent.com/20221326/145188920-60ae5dea-a215-4cd0-ae8c-1c815d4c4c17.png"> | ![screenshot-8](https://user-images.githubusercontent.com/20221326/145189287-9ba00729-bdeb-4ef7-a690-8b45f02ff7f1.png) |

### Readme Update Made

[#66bc3cb](https://github.com/ampproject/amp-wp/pull/6774/commits/66bc3cbcd2ec50b2ffbeeda274f4abdcd8ff3d83) references the new screenshots added for Site Preview and Settings 
 

## Checklist

- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

maitreyie-chavan committed:
Add screenshots to showcase new UI updates

Recommending to replace the screenshots currently used for `Site Preview` in onboarding wizard and `Settings Page` to incorporate the new site scan tab

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ampproject/amp-wp?pullRequest=6774) <br/>All committers have signed the CLA.

maitreyie-chavan committed:
Use new screenshots for Site Preview and Settings

This commit references the newly added screenshot files to showcase the updated Site Preview UI and Site Scan tab in the Settings page since v2.2

westonruter committed:
Rename/replace screenshots to have consecutive filenames

westonruter committed:
Squoosh images to resize and compress
